### PR TITLE
Add typescript example  Azure App Service 😊

### PR DESCRIPTION
**Pull request by guan840912**
Add one typescript azure  example to demonstrating below CDK for Terraform features:

Azure App Service

### CODE REVIEW

File: examples/typescript/azure-app-service/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "azure-app-service-docker",
```
Comment (skorfmann) [inline examples/typescript/azure-app-service/package.json:2]: please name it `@examples/azure-app-service-docker` - see https://github.com/hashicorp/terraform-cdk/blob/7faebbe7ace248377c3ca35c7d40697e797f14e2/CONTRIBUTING.md#examples

**Review by skorfmann — CHANGES_REQUESTED** [review]
Cool - thanks for the example @guan840912 🙌 

Please see the comment for the name change, other than that it's good to go 👍

File: examples/typescript/azure-app-service/package.json (hunk 1 above)
Comment (guan840912) [reply examples/typescript/azure-app-service/package.json]: No problem !!! Sorry my mistake 😊

Comment (skorfmann) [reply examples/typescript/azure-app-service/package.json]: No worries, that's something which got introduced very recently

**Review by skorfmann — APPROVED** [review]

**Pull request merged**